lib/_stream_writable.js

```diff
@@ -155,11 +155,10 @@ Writable.prototype.pipe = function() {
 };
 
 
-function writeAfterEnd(stream, cb) {
-  var er = new Error('write after end');
-  // TODO: defer error events consistently everywhere, not just the cb
-  stream.emit('error', er);
-  process.nextTick(cb, er);
+function writeAfterEndErr(stream, cb) {
```

Comment:
I'm probably too conscious about what function names says at first glance :) `writeAfterEnd()` gave me the impression it should perform a write operation after it has ended. Appending **err** to the name makes it explicit there's an error involved in what's going to happen.

If you see it as superfluous, I'll revert the name change - just say the word :)